src/pykeen/training/training_loop.py

```diff
@@ -607,6 +620,8 @@ def _train(  # noqa: C901
                     if only_size_probing and evaluated_once:
                         break
 
+                    callback.post_batch(epoch=epoch, batch=batch)
```

Comment:
@mberr should we have a `callback.pre_batch()`?

Reply:
I would wait until we need it :slightly_smiling_face:

Reply:
Agreed